### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `Milestone 1`
**In this milestone.**
I added the following:

- HTML (2 sections the header and headline section)
- CSS (styled the header and headline section)
- LINTERS (.stylintrc.json and .hintrc files)
- MARKDOWN
- README (A detailed README.md file)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  font-size: 30px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+}
```
Comment (Emmyn5600) [inline style.css:52]: - [ ] DRY stands for `Don't Repeat Yourself`, however you are repeating ourselves in your code, `display:flex , align-items`  `justify-content: space-evenly` `align-items: center` `flex-wrap: wrap`.

How do you keep CSS dry?

   DRY CSS comes down to 3 things:
- Group reusable css properties together.
- Name these groups logically.
- Add your selectors to the various css groups.

You can make a class and call it `.d-flex` to be called wherever you need that action

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @gabrielcoder247 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

       * #### Kindly, Modify the PR to make it descriptive, use past tense to make the descriptive PR

       * #### Please check the comment under the review
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

DRY CSS was a requirement in 1.0, but it's no longer the case in 2.0, the CR wasn't aware of this and I'll let them know! sorry for that,
However, having a descriptive PR description is required, and I see you've updated it, well done 👏

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <section class="headline">
+            <div class="name">
+                <h1>Gabriel <br> Nwachukwu</h1>
+            </div>
+            <div class="main">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+
+            </div>
+            <button class="collaboration" type="button">
+                start collaboration
```
Comment (ahmetbozaci) [inline index.html:34]: ### OPTIONAL

- [ ] The button should and  the button styling: Active, Hover and Pressed - [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: Approved  🥳   🎉🎉🎉

### Hello @gabrielcoder247  ✋🏼 Nice done with working on last review. I am impressed. Your project is great. I approve it.There is nothing else to say other than... it's time to merge it and delete the branch. You can submit this project in your dashboard.


### Optional suggestions

#### These are optional but highly recommended.
- [ ] _**Check the comments under the review.**_



### Cheers and Happy coding! 💻 👏
aproved

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**